mmdet/apis/ote/apis/detection/train_task.py

```diff
@@ -170,6 +176,11 @@ def train(self, dataset: DatasetEntity, output_model: ModelEntity, train_paramet
     def save_model(self, output_model: ModelEntity):
         buffer = io.BytesIO()
         hyperparams_str = ids_to_strings(cfg_helper.convert(self._hyperparams, dict, enum_to_str=True))
+        if hasattr(self._model, 'bbox_head') and hasattr(self._model.bbox_head, 'anchor_generator'):
+            if getattr(self._config.model.bbox_head.anchor_generator, 'reclustering_anchors', False):
+                hyperparams_str['anchor_heights'] = self._model.bbox_head.anchor_generator.heights
```

Comment:
`hyperparams` are configurable parameters. IMO, you should not change them here.

Reply:
Fixed